src/banana/data/genpdf/__init__.py

```diff
@@ -0,0 +1,200 @@
+import copy
+import pathlib
+import shutil
+
+import lhapdf
+import numpy as np
+from eko import basis_rotation as br
+
+from ... import toy
+from . import export, load, project
+
+
+def generate_pdf(
+    name, labels, parent_pdf_set=None, members=False, info_update=None, install=False
+):
+    """
+    Generate a new PDF from a parent PDF with a set of flavors.
+
+    If parent_pdf_set is the name of an available PDF set, it will be used as parent. In order to use
```

Comment:
Can you please cut the line at 100 characters? just to please pylint :upside_down_face:

Reply:
Maybe try with an auto formatter (at least in vim is possible to select some lines and format them, and you can set a maximum length in configurations, most likely something like this is available for any editor).